src/plugins/mdraid.c

```diff
@@ -173,45 +178,75 @@ static GHashTable* parse_mdadm_vars (gchar *str, gchar *item_sep, gchar *key_val
 static BDMDExamineData* get_examine_data_from_table (GHashTable *table, gboolean free_table, GError **error) {
     BDMDExamineData *data = g_new0 (BDMDExamineData, 1);
     gchar *value = NULL;
+    gchar *first_space = NULL;
+    BSError *bs_error = NULL;
+    struct tm tm;
+    char time_str[20];
 
-    data->level = g_strdup ((gchar*) g_hash_table_lookup (table, "MD_LEVEL"));
+    data->level = g_strdup ((gchar*) g_hash_table_lookup (table, "Raid Level"));
 
-    value = (gchar*) g_hash_table_lookup (table, "MD_DEVICES");
+    value = (gchar*) g_hash_table_lookup (table, "Raid Devices");
     if (value)
         data->num_devices = g_ascii_strtoull (value, NULL, 0);
     else
         data->num_devices = 0;
 
-    data->name = g_strdup ((gchar*) g_hash_table_lookup (table, "MD_NAME"));
+    data->name = g_strdup ((gchar*) g_hash_table_lookup (table, "Name"));
 
-    value = (gchar*) g_hash_table_lookup (table, "MD_ARRAY_SIZE");
-    if (value)
-        data->size = bd_utils_size_from_spec (value, error);
+    value = (gchar*) g_hash_table_lookup (table, "Array Size");
+    if (value) {
+        first_space = strchr (value, ' ');
+        if (first_space)
+            *first_space = '\0';
+        if (value && first_space)
+            /* Array Size is in KiB */
+            data->size = g_ascii_strtoull (value, NULL, 0) * 1024;
+          }
```

Comment:
The ``}`` has wrong indentation. And while changing that, please move the ``else`` on the same line.